lib/helpers.js

```diff
@@ -153,5 +174,14 @@ export function buildOutputText(output) {
     outputText += `Files that stayed the same size 🤷‍:\n\n${reportTable(same)}\n\n`;
   }
 
+  if (totals) {
+    const totalsRows = [
+      { file: 'js', ...totals.js },
+      { file: 'css', ...totals.css },
+    ];
+
+    outputText += `Total Sizes 📊:\n\n${reportTable(totalsRows)}`;
```

Comment:
One possible drawback of reusing `reportTable` here is that it prefixes these totals with the sign `+`.

I accepted the tradeoff, so I could reuse the function. If we want to avoid adding `+` for totals, I can look at adding an option to `reportTable`.

Reply:
Doesn't really bother me.

If we want to be a little lazy about it, `reportTable(totalsRows).replace('+','')` could be an easy win / first pass here, depending on how the maintainer feels about it.

Reply:
I've already had 1 other comment about getting rid of them from my test repo, so it'll probably happen.

Programmers man ... they notice that stuff, like, _straight away_ 😄